### PR TITLE
MAUI: Improve Sample Styling on Android

### PR DESCRIPTION
**Pull request by amweis226**
# Description

Change app theme colors to account for dark theme and fix entry color on Android

## Type of change

- [ ] Bug fix
- [ ] New sample implementation
- [x] Sample viewer enhancement
- [ ] Other enhancement

**Platforms tested on**:

- [x] MAUI Android

## Checklist

- [x] Runs and compiles on all active platforms
- [x] Legacy platforms still compile and run (if applicable)
- [x] Branch is up to date with the latest main/v.next
- [x] All merge conflicts have been resolved
- [x] Self-review of changes
- [x] There are no warnings related to changes
- [x] No unrelated changes have been made to any other code or project files
- [x] Codemaid and XAML styler extensions have been run on every changed file
- [x] There is no leftover commented code

### CODE REVIEW

**Review by ZackAllen — CHANGES_REQUESTED** [review]
No track lines

Track lines dont show by default in Android Light mode.
The button styles are better than before but I still think we should change them to feature text colors that match labels.

**Review by ZackAllen — APPROVED** [review]

**Review by pMaske — APPROVED** [review]
Looks good @amweis226 :-)

File: src/MAUI/Maui.Samples/Resources/Styles/Styles.xaml [hunk 1]
```diff
@@ -123,27 +124,26 @@
...
         <Setter Property="Padding" Value="14,10" />
         <Setter Property="VisualStateManager.VisualStateGroups">
             <VisualStateGroupList>
                 <VisualStateGroup x:Name="CommonStates">
                     <VisualState x:Name="Normal" />
                     <VisualState x:Name="Disabled">
                         <VisualState.Setters>
-                            <Setter Property="TextColor" Value="{AppThemeBinding Light={StaticResource DarkGray}, Dark={StaticResource LightGray}}" />
-                            <Setter Property="BackgroundColor" Value="{AppThemeBinding Light={StaticResource LightGray}, Dark={StaticResource DarkGray}}" />
+                            <Setter Property="TextColor" Value="{AppThemeBinding Light={StaticResource LightGray}, Dark={StaticResource DarkGray}}" />
```
[the remark this answers is not in the document]
Comment (amweis226) [reply src/MAUI/Maui.Samples/Resources/Styles/Styles.xaml]: Noting that we resolved this together over a video meeting

File: src/MAUI/Maui.Samples/Resources/Styles/Styles.xaml [hunk 2]
```diff
@@ -367,9 +368,9 @@
     </Style>
 
     <Style TargetType="Slider">
-        <Setter Property="MinimumTrackColor" Value="{AppThemeBinding Light={StaticResource Primary}, Dark={StaticResource White}}" />
-        <Setter Property="MaximumTrackColor" Value="{AppThemeBinding Light={StaticResource LightGray}, Dark={StaticResource DarkGray}}" />
-        <Setter Property="ThumbColor" Value="{AppThemeBinding Light={StaticResource Primary}, Dark={StaticResource White}}" />
+        <Setter Property="MinimumTrackColor" Value="{OnPlatform Default={AppThemeBinding Light={StaticResource Primary}, Dark={StaticResource White}}, Android={AppThemeBinding Light={StaticResource Primary}, Dark={StaticResource Primary}}}" />
```
[the remark this answers is not in the document]
Comment (amweis226) [reply src/MAUI/Maui.Samples/Resources/Styles/Styles.xaml]: Noting that we resolved this together over a video meeting

**Pull request merged**